code/game/objects/items/storage/holsters.dm

```diff
@@ -0,0 +1,119 @@
+
+/obj/item/storage/belt/holster
+	name = "shoulder holster"
+	desc = "A rather plain but still badass looking holster with a single pouch that can hold a small firearm."
+	icon_state = "holster"
+	item_state = "holster"
+	alternate_worn_layer = UNDER_SUIT_LAYER
+
+/obj/item/storage/belt/holster/equipped(mob/user, slot)
+	. = ..()
+	if(ishuman(user) && slot == ITEM_SLOT_BELT)
```

Comment:
why only humans

Reply:
This calls on the children of humans as well. Here is me flipping as an Andriod
https://www.youtube.com/watch?v=2OWqRP5VFMo&feature=youtu.be

Reply:
Unless of course, you want goliaths and slimes to be able to flip even though they don't have belt slots.

Reply:
I don't understand why you need the ishuman check specifically.

If they have the slot ,they should be able to flip. so the slot check should be sufficient.

Reply:
If it bugs you that much ill remove it.